components/data-apps/general-design-guide/index.md

```diff
@@ -0,0 +1,233 @@
+---
+title: Data Apps General Design Guide
+permalink: /components/data-apps/general-design-guide/
+---
+
+* TOC
+{:toc}
+
+This guide will help you quickly create a user-friendly data app in Keboola. We cover essentials like theming, headers, storage integration, 
+and more to streamline your setup. With clear, step-by-step instructions, you’ll be able to build an intuitive, visually appealing app. 
+
+## Theming
+There are two options for setting theming:
+
+1. **Keboola Data Apps Interface:** Choose from predefined or custom themes directly in the Keboola interface. See more in [the documentation](/components/data-apps/#theming).
+2. **Configuration File:** Use the settings in the `config.toml` file, located in the `.streamlit` folder.
+
+{: .image-popup}
+![Screenshot - Streamlit Folder](/components/data-apps/general-design-guide/pic1.png)
+
+```
+[theme]
+font="sans serif"
+textColor="#222529"
+backgroundColor="#ffffff"
+base="light"
+primaryColor="#1f8fff"
+secondaryBackgroundColor="#edf0f5"
+```
+
+## Header
+
+### Logo
+
+Store the logo PNG image in the `/static/` repository folder (created automatically upon deployment in Keboola).
+
+```
+LOGO_IMAGE_PATH = os.path.abspath("./app/static/keboola.png")
+```
+
+Position the logo on the left and disable the full-screen view for the image.
+
+```
+st.image(LOGO_IMAGE_PATH)
+hide_img_fs = '''
+        <style>
+        button[title="View fullscreen"]{
+            visibility: hidden;}
+        </style>
+        '''
+st.markdown(hide_img_fs, unsafe_allow_html=True)
+```
+
+## Body
+
+### Set Up Caches
+
+Set the `ttl` parameter based on your app's needs.
+
+```
+# Fetching data 
+@st.cache_data(ttl=7200,show_spinner=False)
+```
+
+### Info Panel
+
+{% include tip.html title="" content="
+Select the table you want to edit. If the data is not up to date, click the Reload Data button. The data freshness is displayed in the right corner.
+" %}
+
+Use `st.info` to provide helpful context or instructions to users.
+
+```
+st.info('Select the table you want to edit. If the data is not up to date, click the Reload Data button. The data freshness is displayed in the right corner.', icon="ℹ️")
+```
+
+### Hide Anchor Links
+
+To hide anchor links, add this function.
+
+```
```

Comment:
tady je pořeba přidat {% raw %}
jako tady https://github.com/keboola/developers-docs/blob/main/integrate/variables/index.md
dělají to dvojité chlupaté závorky